tools/generator/test_AnDOGenerator.py

```diff
@@ -0,0 +1,87 @@
+import unittest
+from tools.generator.AnDOGenerator import *
+
+
+class Test_AnDOSesID(unittest.TestCase):
+
+    def setUp(self):
+        self.date = '20000101'
+        self.sesNumber = '100'
+        self.customField = 'test'
```

Comment:
should be `customSesField` instead of `customField`, right?

Reply:
Yes, I did replace it in the complete code.